Finalized release notes for v1.2.3 (#1244)
nautobot/nautobot: nautobot/docs/release-notes/version-1.2.md

### Fixed

- [#313](https://github.com/nautobot/nautobot/issues/313) - REST API documentation now correctly shows that `status` is a required field.

> - [#477](https://github.com/nautobot/nautobot/issues/477) - Fixed `nautobot.utilities.filters.BaseFilterSet.FILTER_DEFAULTS` does not allow `models.TextField` to use multi-value filtering. Models using `TextField`s will now be able to `MultiValCharFilter` in filter classes on those fields.

Reviewer: ```suggestion
- [#477](https://github.com/nautobot/nautobot/issues/477) - Model `TextField`s are now correctly mapped to `MultiValueCharFilter` in filter classes.
```
